front/src/components/celebrities/celebrities.component.vue

```diff
@@ -0,0 +1,5 @@
+<template src="./celebrities.component.html"></template>
+
+<script src="./celebrities.component.js"></script>
+
+<style scoped src="./celebrities.component.css"></style>
```

Comment:
it is not component, it is view
so, celebrities.view.html/css/js/vue

Reply:
Ok, I'll fix that